gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -385,6 +392,89 @@ nnsconf_loadconf (gboolean force_reload)
   return TRUE;
 }
 
+/** @brief Public function defined in the header */
+gboolean
+nnsconf_loadconf_env (void)
```

Comment:
Suggestion: to reduce duplicated code, how about adding a param in old function?
```
gboolean
nnsconf_loadconf (gboolean force_reload, gboolean from_env)
```

Reply:
OK I will fix this